ChildProject/pipelines/samplers.py

```diff
@@ -711,74 +713,65 @@ def _segment_scores(self, recording):
         ).reset_index()
         windows["recording_filename"] = recording["recording_filename"]
 
-        if self.metric == "ctc":
+        if self.metric == "turns":
             if "lena_conv_turn_type" in segments.columns:
                 # NOTE: This is the equivalent of CTC (tab1) in rlena_extract.R
                 segments["is_CT"] = segments["lena_conv_turn_type"].isin(
                     ["TIFR", "TIMR"]
                 )
             else:
                 segments.sort_values(["segment_onset", "segment_offset"])
-                segments = segments[
-                    segments["speaker_type"].isin(["FEM", "MAL", "CHI"])
-                ]
-                segments["duration"] = (
-                    segments["segment_offset"] - segments["segment_onset"]
-                )
+                segments = segments[segments["speaker_type"].isin(self.speakers)]
                 segments["iti"] = segments["segment_onset"] - segments[
```

Comment:
Right, so as to leave a note on the logic:
- for LENA, we just follow LENA's counts
- for ACLEW, we select turns concerning the speakers of choice

Reply:
Now mentioned in the docs